### PR TITLE
@orta=>Hide sync notification badge & update sync view controller copy

### LINKED ISSUE #140: Exclamation mark noting sync is needed does not go away after syncing
Exclamation mark symbol noting that a sync is needed does not disappear after sync has occurred if user has updated to new version without logging out 

img_0071

### PR DESCRIPTION
**Pull request by sarahscott**
Since the CMS notification badge has been removed from the new settings for the time being, this will make sure it's hidden for new installs.

Fixes #140 & #139

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
&nbsp; | 2 Warnings
    ------------- | ------------
        :warning: | PR is classed as Work in Progress
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by sarahscott** [comment]
@orta greeeeen

File: docs/CHANGELOG.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 upcoming:
   version: 2.5.0
   notes:
+    - (dev) Updated sync view controller copy [sarah]
+    - (dev) Removed sync notification badge from settings icon [sarah]
```
Comment (orta) [inline docs/CHANGELOG.yml]: this was useful, it should go back in before you ship to the app store

**Comment by orta** [comment]
:+1:

**Pull request merged**